web/crux-ui/e2e/with-login/registry.spec.ts

```diff
@@ -93,3 +93,30 @@ test("Unchecked registry shouldn't search images", async ({ page }) => {
 
   await page.locator('button:text-is("Add")').click()
 })
+
+test('Image list should be visible', async ({ page }) => {
+  const registryName = 'TEST REGISTRY IMAGE LIST'
+
+  await page.goto(TEAM_ROUTES.registry.list())
+
+  await page.locator('text=Add').click()
+  await expect(page.locator('h4')).toContainText('New registry')
+
+  await page.locator('input[name=name] >> visible=true').fill(registryName)
+  await page.locator('form >> text=Docker Hub').click()
+  await expect(await page.locator('label[for=imageNamePrefix]')).toContainText('Organization name or username')
+  await page.locator('input[name=imageNamePrefix]').fill('library')
+
+  await page.screenshot({ path: screenshotPath('registry_new_filled'), fullPage: true })
```

Comment:
This screenshot name is also used in the same file elsewhere which would overwrite it. I think you could remove the screenshot because we already have one from the create registry test.